### ISSUE TITLE
[dev.icinga.com #7974]  Create a mechanism that allows users to jump to errors

### ISSUE THREAD

**Issue by icinga-migration** [Icinga/icingaweb2#1244]
This issue has been migrated from Redmine: https://dev.icinga.com/issues/7974

**Created by elippmann on 2014-12-03 11:53:58 +00:00**

Assignee: _(none)_
Status: _Closed_  (closed on _2016-02-19 16:06:32 +00:00_)
Target Version: _Accessibility_
Last Update: _2016-02-19 16:06:32 +00:00 (in Redmine)_

---

http://www.w3.org/TR/WCAG20-TECHS/general.html#G139 wrote:
> The objective of this technique is to help users find input errors where the information supplied by the user is not accepted. This includes fields with missing required information and fields with incorrect information. When users enter data input that is checked, and input errors are detected, a link to that error is provided so that the user does not have to search for it. One approach is to use server-side validation, and to re-display the form (including any previously entered data), and a text description at the top of the page that indicates the fact that there was an input error, describes the nature of the problem, and provides a link the field(s) with a problem.

**Acceptance Criteria:**

-   Fill out a form, deliberately leaving a required (mandatory) field blank, and make an input error on another field and submit the form.
-   Check that a text message is provided that identifies the field that is missing required data.
-   Check that a text message is provided that identifies the field with the input error.
-   Check that there is a link to each field that is missing required data from the missing data message
-   Check that there is a link to the list of errors from the error message.

-   If \#2 is true, then \#4 is true.
-   If \#3 is true, then \#5 is true.
